moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js

```diff
@@ -236,7 +239,21 @@ const SignalProjectTable = () => {
     {
       title: "Task order",
       field: "task_order",
-      // placeholder for task order issue
+      customEdit: "taskOrders",
+      emptyValue: "blank",
```

Comment:
The other emptyValues on the table are now `emptyValue: "-"`

Reply:
Thanks, I had a feeling I guessed wrong on what the decision was for default empty value text I and I really appreciate this correction. I'll fix this in a commit.